pkg/commands/engine/object.go

```diff
@@ -111,6 +111,40 @@ The JSON objects can be in either the GenericObjectProperties format or the Gene
 		},
 	}
 
+	var publishObjectCmd = &cobra.Command{
+		Use:     "publish <object-id>",
+		Args:    cobra.ExactArgs(1),
+		Short:   "Publish a generic object, like to make a sheet public",
+		Long:    "Publish a generic object, like to make a sheet public",
+		Example: "corectl object publish OBJECT-ID -a APP-ID",
+
```

Comment:
You could mark the introduced commands as experimental if you want by adding the following:
```go
   Annotations: map[string]string{
      "x-qlik-stability": "experimental",
   }
```

Reply:
where does that get added?

Reply:
oh nevermind, I get it

Reply:
circle ci doesn't like the Annotations on the commands.